webvi-systemlink-plugin/WebApp.gcomp/index.gviweb

```diff
@@ -1,12 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
WebVI updated to match style of other examples
![image](https://user-images.githubusercontent.com/1588923/87608052-86762200-c6c4-11ea-8c30-f7efbd0c2b7b.png)